Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -521,12 +588,14 @@ def test_get_search_by_username(self):
         assert response.json()["ReadOnly"] == 0
         assert response.json()["account_id"] == 30580
 
+
 class Test_AllEmailTest(testCase):
 # # # # # # # #
-# EMAIL  TEST #
+# EMAIL TEST  #
 # # # # # # # #
 
-#    Verify that a student can get a list of the emails for all members in the activity
+#    Verify that a student can get a list of the emails for all members in the 
+#    activity.
```

Comment:
A student member, or non-member?  
I'm not sure if students are supposed to be able to see members or not.  At least, I don't see how this info is made visible on 360 (though it seems useful and I think OK to show it).

Reply:
A student member. I think I'll just leave the test as is, because it might be useful for a feature in the future.